src/showComments.js

```diff
@@ -0,0 +1,38 @@
+import { retreiveData, submitComment } from './involvementAPI.js';
+
+const comments = document.querySelector('.comment-list');
+const commentsNumber = document.querySelector('.comments-number');
+const inputUsername = document.getElementById('input-username');
+const publishCommentButton = document.getElementById('publish-comment-button');
+const inputDescription = document.getElementById('input-description');
+
+const displayComments = async () => {
+  const comment = await retreiveData();
+  comments.innerHTML = '';
+  commentsNumber.innerHTML = '';
```

Comment:
This line is unnecessary because `element.innerHTML` will overwrite the existing content, so in next line the content will get's overwritten